docs/source/delta-spark-connect.md

```diff
@@ -0,0 +1,121 @@
+---
+description: Learn about Delta Connect - Spark Connect in Delta.
+---
+
+# Delta Connect (or Spark Connect in Delta)
+
+.. note:: This feature is available in preview in <Delta> 4.0 and above.
+
+<Delta> Connect adds [Spark Connect](https://spark.apache.org/docs/latest/spark-connect-overview.html) support to Scala and Python APIs of <Delta> Lake for Apache Spark. Spark Connect is a new project released in Apache Spark 4.0 that adds a decoupled client-server infrastructure which allows remote connectivity from Spark from everywhere. <Delta> Connect makes the `DeltaTable` interfaces compatible with the new Spark Connect protocol.
+
+## Motivation
+
+<Delta> Connect is expected to bring the same benefits as Spark Connect:
+
+1. Easier upgrading to more recent versions of Spark and <Delta>, as the client interface is completely decoupled from the server.
+2. Simpler integration of Spark and <Delta> with developer tooling. IDEs no longer have to integrate with the full Spark and <Delta> implementation, and instead can integrate with a thin-client.
+3. Support for languages other than Java/Scala and Python. Clients "merely" have to generate Protocol Buffers and therefore become simpler to implement.
+4. Spark and <Delta> will become more stable, as user code is no longer running in the same JVM as Spark's driver.
+5. Remote connectivity. Code can run anywhere now, as there is a gRPC layer between the user interface and the driver.
+
+## Supported Delta Connect operations
```

Comment:
Move this section to the end and rename it as "Preview Limitations"

Reply:
And make it a shorter list of things that are known to not work

Reply:
@tdas How can we make it shorter? I already shortened the list (you can do many things with `merge` and `optimize`), shall I omit `createIfNotExists` and combine `restoreToVersion` and `restoreToTimestamp` into `restore`?

Reply:
@tdas Well for Preview we only support `forName`, and `forPath`. In the official 4.0 release we plan to support all operations in https://github.com/delta-io/delta/blob/master/python/delta/tables.py for Delta Connect.

Does this answer your question? I made a board in the doc.